tests/test_xml.py

```diff
@@ -6,22 +6,14 @@
 import stat
 import tempfile
 
-from xml.etree import ElementTree
-try:
-    from xml.etree.ElementTree import ParseError
-except ImportError:
-    from xml.parsers.expat import ExpatError as ParseError
-
```

Comment:
I don't understand how this change can affect anything. The imports I removed are not used in this test module at all, neither ElementTree, nor ParseError.